azurelinuxagent/common/protocol/wire.py

```diff
@@ -247,7 +260,16 @@ def _build_health_report(incarnation, container_id, role_instance_id,
                        container_id,
                        role_instance_id,
                        status,
-                       detail)
+                       detail,
+                       users_element)
+    return xml
+
+
+def serialize_users(users):
+    xml = ''
+    if users is not None and len(users) != 0:
+        for user in users:
+            xml = xml + '<User>' + user[0] + '</User>'
```

Comment:
Please use the built in serialization to handle this.  No need to re-invent it.

Reply:
done